src/prefect/engine/serializers.py

```diff
@@ -5,9 +5,9 @@
 import cloudpickle
 
 
-class Serializer:
+class BytesSerializer:
```

Comment:
I don't think this name makes sense - all serializers work to/from bytes. The qualifier on a `Serializer` should be the protocol used (e.g. `PickleSerializer`, `JsonSerializer`, `CSVSerializer`, ...) not the input/output type (which is always bytes.

Reply:
Switched to `PickleSerializer` as discussed in core release planning today in https://github.com/PrefectHQ/prefect/pull/2755/commits/a58b26bb65424379b6b7b838bbc7ed4b3984d09e

Reply:
Thanks!